fdbserver/RestoreLoader.actor.cpp

```diff
@@ -125,29 +123,19 @@ void handleRestoreSysInfoRequest(const RestoreSysInfoRequest& req, Reference<Res
 	req.reply.send(RestoreCommonReply(self->id()));
 }
 
-void handleSetApplierKeyRangeVectorRequest(const RestoreSetApplierKeyRangeVectorRequest& req,
-                                           Reference<RestoreLoaderData> self) {
-	TraceEvent("FastRestore")
-	    .detail("Loader", self->id())
-	    .detail("SetApplierKeyRangeVector", req.rangeToApplier.size());
-	// Idempodent operation. OK to re-execute the duplicate cmd
-	if (self->rangeToApplier.empty()) {
-		self->rangeToApplier = req.rangeToApplier;
-	} else {
-		ASSERT(self->rangeToApplier == req.rangeToApplier);
-	}
-	req.reply.send(RestoreCommonReply(self->id()));
-}
-
 ACTOR Future<Void> _processLoadingParam(LoadingParam param, Reference<RestoreLoaderData> self) {
 	// Q: How to record the  param's fields inside LoadingParam Refer to storageMetrics
 	TraceEvent("FastRestore").detail("Loader", self->id()).detail("StartProcessLoadParam", param.toString());
 	ASSERT(param.blockSize > 0);
 	ASSERT(param.offset % param.blockSize == 0); // Parse file must be at block bondary.
+	ASSERT(self->kvOpsPerLP.find(param) == self->kvOpsPerLP.end());
+	// NOTE: map's iterator is guaranteed to be stable, but pointer may not.
+	// state VersionedMutationsMap* kvOps = &self->kvOpsPerLP[param];
+	self->kvOpsPerLP.insert(std::make_pair(param, VersionedMutationsMap()));
```

Comment:
```suggestion
	self->kvOpsPerLP.insert({param, VersionedMutationsMap()});

```
Alternatively, `self->kvOpsPerLP.emplace(param, VersionedMutationsMap());` is even shorter.